Common/UnitDefinitions/ForceChangeRate.json

```diff
@@ -33,6 +33,32 @@
           "Abbreviations": [ "N/s" ]
         }
       ]
+    },
+	{
+      "SingularName": "PoundForcePerMinute",
+      "PluralName": "PoundsForcePerMinute",
+      "FromUnitToBaseFunc": "x*4.4482216152605095551842641431421/60",
+      "FromBaseToUnitFunc": "x/4.4482216152605095551842641431421*60",
+      "Prefixes": [ "Deca", "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lbf/min" ]
```

Comment:
Should there be a `kipf` abbreviation for Kilo prefix for these two new units?

`"AbbreviationsForPrefixes": { "Kilo": ["kipf/s"] }`

Reply:
Yes, you're right, i copy/paste the json from the force unit, and in this one, PoundForce and KiloPoundForce are two differents units. So i changed it, added Kilo prefixe in PoundForce, added abbreviations for this prefixe and removed the KiloPoundForce  unit definition.
And of course I fixed ForceChangeRate.json to add abbreviations for kilo prefixe.